src/Famix-MetamodelBuilder-Core/FmxBanana.class.st

```diff
@@ -0,0 +1,8 @@
+"
+I'm a banana
```

Comment:
@badetitou Is there a reason you committed this class?

Reply:
I wanna say no